Examples/iOS/SequencerDemo/SequencerDemo/Conductor.swift

```diff
@@ -90,6 +90,7 @@ class Conductor {
         sequence.setTempo(100)
         sequence.play()
     }
+    
```

Comment:
The hounds don't like white space at the end of lines, or lines that just contain some spaces when they could be returns.  Its complaining about horizontal white space, not vertical.  Its a minor thing anyway.